xml/ha_maintenance.xml

```diff
@@ -37,6 +37,50 @@
  </info>
  <sect1 xml:id="sec-ha-maint-shutdown-node">
   <title>Implications of Taking Down a Cluster Node</title>
+  <para>
+   In &sls; &hasi; 15 SP2, the way how cluster services were started and
+   stopped has been changed. &suse; recommends to use the &crmshell;. The
+   old commands with <command>systemctl</command> are still available, but
+   are recommended for experienced users only.
+   For details, refer to the &suse; blog article <link
+    xlink:href="https://www.suse.com/c/suse-high-availability-cluster-services-how-to-stop-start-or-view-the-status/"
+   />.
+  </para>
+  <para>
+   The recommended way of starting, stopping, or viewing the status is:
+  </para>
+
+  <variablelist>
+   <varlistentry>
+    <term>Starting the cluster services on one node</term>
+    <listitem>
+     <screen>crm cluster start</screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Stopping the cluster services on one node</term>
+    <listitem>
+     <screen>crm cluster stop</screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Restarting the cluster services on one node</term>
+    <listitem>
+     <screen>crm cluster restart</screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Viewing the status of the cluster stack on one node</term>
+    <listitem>
+     <screen>crm cluster status</screen>
+    </listitem>
+   </varlistentry>
+  </variablelist>
```

Comment:
I suggest switching around the commands and the explanations of the commands.

So the [term] would be the command, in [screen] tags, and the [listitem] would be the explanation of what it does.

Additionally I recommend removing the definite articles in the explanations:
• stopping cluster services
• restarting cluster services
etc. This is only a _suggestion_ though!

Reply:
Good idea! I'll switch it as adivsed. :+1: